core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -422,6 +403,32 @@ public static boolean useEpoll() {
      * {@code -Dcom.linecorp.armeria.useOpenSsl=false} JVM option to disable it.
      */
     public static boolean useOpenSsl() {
+
+        if (USE_OPENSSL == null) {
```

Comment:
We prefer ['early-return' style code](https://github.com/line/armeria/blob/master/CONTRIBUTING.md#prefer-early-return-style). Could you change the code as the the following code style?
```java
if(useOpenSsl != null) {
	return useOpenSsl;
}

useOpenSsl = getBoolean("useOpenSsl", OpenSsl.isAvailable(), value -> OpenSsl.isAvailable() || !value);
...
```

Reply:
OK. I applied it. Thank you 😄